src/Symfony/Component/Console/Helper/ProcessHelper.php

```diff
@@ -27,12 +27,11 @@ class ProcessHelper extends Helper
     /**
      * Runs an external process.
      *
-     * @param OutputInterface      $output    An OutputInterface instance
-     * @param string|array|Process $cmd       An instance of Process or an array of arguments to escape and run or a command to run
-     * @param string|null          $error     An error message that must be displayed if something went wrong
-     * @param callable|null        $callback  A PHP callback to run whenever there is some
-     *                                        output available on STDOUT or STDERR
-     * @param int                  $verbosity The threshold for verbosity
+     * @param string|array|\Process $cmd       An instance of Process or an array of arguments to escape and run or a command to run
```

Comment:
wrong `\` on `Process`, class is correctly imported

Reply:
Fixed: https://github.com/symfony/symfony/compare/master...TomasVotruba:phpdos-removal-master#diff-425e56e3b866cb257f587805f7b2ee54R30